gradle.properties

```diff
@@ -1,4 +1,4 @@
-projectVersion=4.1.1-SNAPSHOT
+projectVersion=4.1.1
```

Comment:
Can you change this to `4.2.0-SNAPSHOT` and change all the `@since` tags you added to `4.2.0` I think there are too many new things now for a patch release. Thanks.

Reply:
done

Reply:
the `@since` tags still reference 4.1.1

Reply:
sorry, i'm doing this between meetings.
fixed the since tags.